_includes/footer.html

```diff
@@ -23,5 +23,9 @@
 <script src="{{base}}/js/materialize.min.js"></script>
 <script src="{{base}}/js/init.js"></script>
 <script src="{{base}}/js/functions.js"></script>
+<script type="text/javascript" id="cookieinfo" src="{{base}}/js/cookieinfo.min.js"
```

Comment:
What do you think of changing the button color to the color of title bar and changing the text to "OK" instead of "Got It?" This would require the following modifications: 
`data-divlinkbg="#f60" data-close-text="OK"`

Reply:
I don't have a problem with "Got It". I think "OK" or "Accept" sounds a bit more serious.

Reply:
Disclaimer, I only copied the first code snippet I saw on https://cookieinfoscript.com/ 😁 
I don't have a problem with either, the yellow button feels like a fresh break from the omnipresent orange :) Anyways, we have 2 days to decide.

Reply:
I think "Got it!" fits the general feel of the project really well.

Reply:
Sorry, @MHerbst, sounds as if you have been overruled ;-) Just no serious folks around here, it seems...